my_list.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(list)
+    @list = list
+  end
```

Comment:
- [ ] Please kindly ensure that the `initialize` method accepts multiple arguments instead of just an array. This will enable the constructor to be called with a list of numbers.

Hint:
```ruby
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```

Reply:
Ok, we will. 
Thanks